lib/workspace/src/state/canvas_state_notifier.dart

```diff
@@ -24,6 +24,7 @@ class CanvasStateNotifier extends StateNotifier<CanvasState> {
   void clearBackgroundImageAndResetDimensions() => state = state.copyWith(
         backgroundImage: Option.none(),
         size: CanvasState.initial.size,
+        resetScale: state.resetScale ? false : true,
```

Comment:
This is kinda a workaround solution. The changes to resetScale are listened to in build method of DrawinngCanvasState. And since ```clearBackgroundImageAndResetDimensions``` wants to reset the canvas it should also reset the scale. The boolean variable acts as a toggle.